domains/certificates/Query.API/API.UnitTests/ContractService/CertificateIssuingContractTests.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using API.ContractService;
+using CertificateValueObjects;
+using FluentAssertions;
+using Xunit;
+
+namespace API.UnitTests.ContractService;
+
+public class CertificateIssuingContractTests
+{
+    /// <summary>
+    /// Contract that runs from 12:00 to 13:00 on Jan 1st 2023
+    /// </summary>
+    private readonly CertificateIssuingContract contract = new()
+    {
+        Id = Guid.NewGuid(),
+        GSRN = "gsrn",
+        GridArea = "gridArea",
+        MeteringPointType = MeteringPointType.Production,
+        MeteringPointOwner = "owner",
+        StartDate = DateTimeOffset.Parse("2023-01-01T12:00:00Z"),
+        EndDate = DateTimeOffset.Parse("2023-01-01T13:00:00Z"),
+        Created = DateTimeOffset.Parse("2023-01-01T12:00:00Z")
+    };
+
+    [Theory]
+    [InlineData("2023-01-01T12:00:00Z", "2023-01-01T13:00:00Z")]
+    [InlineData("2023-01-01T11:00:00Z", "2023-01-01T12:00:01Z")]
+    [InlineData("2023-01-01T11:00:00Z", "2023-01-01T12:00:00.0001Z")]
+    [InlineData("2023-01-01T12:59:59Z", "2023-01-01T14:00:00Z")]
+    [InlineData("2023-01-01T12:59:59.999Z", "2023-01-01T14:00:00Z")]
+    public void has_overlap(string start, string end) =>
```

Comment:
I do like the snake_case method names in test, but i think we should align in the team on how we are naming our tests. Right now we are using different conventions

Reply:
I agree. Do you want to take the lead on landing on a decision?

Reply:
Yes. I'll put it in my calendar after my vacation. :)